ui/packages/catalog-ui-search/src/main/webapp/component/sharing/sharing.view.js

```diff
@@ -0,0 +1,326 @@
+/**
```

Comment:
Reviewers may be particularly interested in this file which contains the new sharing component